src/site/content/en/blog/streams/index.md

```diff
@@ -460,20 +460,20 @@ property returns the desired size to fill the controlled stream's internal queue
 ### The `queuingStrategy`
 
 The second, likewise optional, argument of the `ReadableStream()` constructor is `queuingStrategy`.
-It is an object that optionally defines a queuing strategy for the stream, which takes two
-parameters:
+It is an object that optionally defines a queuing strategy for the stream, which takes one
+parameter:
 
-- `highWaterMark`: A non-negative number indicating the high water mark of the stream using this queuing strategy.
-- `size(chunk)`: A function that computes and returns the finite non-negative size of the given chunk value.
-  The result is used to determine backpressure, manifesting via the appropriate `ReadableByteStreamController.desiredSize` property.
+- `highWaterMark`: A non-negative number of bytes indicating the high water mark of the stream using this queuing strategy.
+  This is used to determine backpressure, manifesting via the appropriate `ReadableByteStreamController.desiredSize` property.
   It also governs when the underlying source's `pull()` method is called.
 
 {% Aside %}
-  You could define your own custom `queuingStrategy`, or use an instance of
-  [`ByteLengthQueuingStrategy`](https://developer.mozilla.org/en-US/docs/Web/API/ByteLengthQueuingStrategy)
-  or [`CountQueuingStrategy`](https://developer.mozilla.org/en-US/docs/Web/API/CountQueuingStrategy)
-  for this object value. If no `queuingStrategy` is supplied, the default used is the same as a
-  `CountQueuingStrategy` with a `highWaterMark` of `1`.
+  A queuing strategy for a readable byte stream does not have a `size(chunk)` function.
```

Comment:
I want to make sure there isn't something else to explain. Why would I expect a `size(chunk)` function?

Reply:
Because all the other stream variants have one to set a custom queuing strategy.

Reply:
maybe add , "like other stream types do"? 

This is the conceptual version of a broken link. If I'm not already familiar with streams, I can't work out for myself why this information is here or why I care.

Reply:
How about:
> Unlike queuing strategies for other stream types, a queuing strategy for a readable byte stream does not have a `size(chunk)` function.

Reply:
I went with this variant in https://github.com/GoogleChrome/web.dev/pull/4713/commits/cdab862037056c1084b3a841c7cb55a421ffe1fa.